src/components/layout/Header.vue

```diff
@@ -36,3 +36,14 @@ export default {
   }
 }
 </script>
+
+<style>
+
+a.nav-link {
+    color: var(--subtle) !important;
+}
+
+a.nav-link:focus, a.nav-link:hover {
```

Comment:
damit überschreiben wir ja wiederum nav-link. Ich würde hier konsistent sein und in `custom.scss` die bootstrap klassen überschreiben

Reply:
Hab ich nicht hinbekommen. Natürlich bootstrap klassen nehmen wenn möglich. Wenn du es mit scss hinbekommst, liebend gerne.

Reply:
$navbar-light-color
$navbar-light-hover-color